bears/ruby/RubySyntaxBear.py

```diff
@@ -14,7 +16,8 @@ class RubySyntaxBear:
     Checks the code with ``ruby -wc`` on each file separately.
     """
     LANGUAGES = {'Ruby'}
-    REQUIREMENTS = {GemRequirement('ruby', '2.1.5')}
+    REQUIREMENTS = {GemRequirement('ruby', '2.1.5'),
```

Comment:
i mostly copy pasted what we had in the gemfiles :) it is probably at that point that i found it relevant, and now since youre adding the system ruby, im asking if we still need the gem ruby :-??